Gemfile

```diff
@@ -1,4 +1,6 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
+# attempt to resolve gem dependencies by ruby version
+ruby RUBY_VERSION
```

Comment:
This will only work with [Bundler 1.13.0+](https://rubygems.org/gems/bundler/versions/1.13.0).

That just means if it fails, you need
* `gem update --system` (to get to at least 1.3.6) / certificate updates to get this to work likely
* `gem update bundler`

You probably already know this and have that implemented somewhere already though.